setup.py

```diff
@@ -26,6 +26,7 @@ def prerelease_local_scheme(version):
     name='tifftools',
     use_scm_version={'local_scheme': prerelease_local_scheme, 'fallback_version': '0.0.0'},
     setup_requires=['setuptools-scm'],
+    install_requires=['pyproj'],
```

Comment:
Please make this some sort of extras_require.  It is convenient to have _no_ dependencies for the core library.

Reply:
Sure, what do we want the extra to be called? Maybe something like `geo` or `geospatial`?

Reply:
Either `geo` or `geotiff`.  I'll note that I don't think pyproj publishes wheels for pypy, so any tests that depend on pyproj will be a pain to run on those environments.

Reply:
And, note the recent commit that removed setup.py in favor of just using pyproject.toml (trying to keep this modern).

Reply:
Gotcha, I'll update my branch with latest master